frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -31,6 +31,15 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
     formatAssessed = '';
   }
 
+  let improvments = 0;
```

Comment:
I just copied what was done above for assessed land. It appears that for all cases of the property slideout it is missing `assessedLand` and `assessedBuilding` as it takes in an `IParcel` or `IBuilding` as a prop not `IProperty`